test-prow-e2e.sh

```diff
@@ -21,4 +21,13 @@ source ./contrib/oc-environment.sh
 kubectl create -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/okd/manifests/0.8.0/0000_30_06-rh-operators.configmap.yaml
 kubectl create -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/okd/manifests/0.8.0/0000_30_09-rh-operators.catalogsource.yaml
 
+echo "Checking for dist files:"
+while [ ! -f ./frontend/public/dist/index.html ]
+do
+ echo "#"
```

Comment:
Second question is probably more important.  Do we want a limited number of retries?

Reply:
I'd be curious to see how many `#` get printed this run, then perhaps do 4x as a limit, in case we want a hard fail.